shop.html

```diff
@@ -1,11 +1,68 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="pl">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>My shop</title>
+    <link rel="stylesheet" href="style.css">
+    
 </head>
 <body>
-    
+    <header>
+        <div class="left">
+            <p class="raz">Razor&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </p>
+            <p class="ab">Wieczko55@gmail.com  <img src="arrow-right-solid.svg" alt="arrow right" height="20px" width="20px"></p>
+        </div>
+        
+        <div class="right">
+            <img src="website.png" alt="website icon image" height="50px" width="50px">
+            <a href="#">Sign up</a>
+            <button>Request Demo</button>
+        </div>
+    </header>
+    <div class="wrapper">
+        <div class="screen">
+            <div class="text">
+                <h2>Agent Login</h2>
+                <p>Hey, Enter your details to get sign in <p> to your account</p>
+                
+            </div>
+            <div class="content">
+                <form action="#" method="post">
+                    <div class="inputbox">
+                        <img src="circle-regular.svg" alt="circle icon" height="20px" width="20px">
+                        <input type="email" required name id>
```

Comment:
nie używaj pustych id do input. Jeśli chcesz już w ten sposób powiązać label z input użyj w input id="email" a w label for="email"